Sources/StreamChatUI/ChatMessageList/ChatChannelVC.swift

```diff
@@ -99,4 +99,27 @@ extension _ChatChannelVC: _ChatChannelControllerDelegate {
     ) {
         messageList.updateMessages(with: changes)
     }
+
+    public func channelController(
+        _ channelController: _ChatChannelController<ExtraData>,
+        didChangeTypingMembers typingMembers: Set<_ChatChannelMember<ExtraData.User>>
+    ) {
+        guard let currentMember = channelController.channel?.membership else {
+            typingIndicatorView.setAnimatedly(hidden: true)
+            log.error("Current member not found in ChatChannelVC.")
+            return
+        }
+
+        let typingMembersWithoutCurrentUser = typingMembers.subtracting([currentMember])
+
+        if typingMembersWithoutCurrentUser.isEmpty {
+            typingIndicatorView.setAnimatedly(hidden: true)
+        } else if typingMembersWithoutCurrentUser.count == 1 {
+            guard let member = typingMembers.first, let name = member.name else { return }
+            typingIndicatorView.content = "\(name) " + L10n.MessageList.TypingIndicator.typingSingular
+        } else {
+            guard let member = typingMembers.first, let name = member.name else { return }
+            typingIndicatorView.content = "\(name) and \(typingMembers.count - 1) " + L10n.MessageList.TypingIndicator.typingPlural
```

Comment:
I would suggest passing the parameters into the localised string. 

We should enable the clients to change the positions of the parameters in the localised string. So they can change from: "Dominik and 3 others are typing" to "Typing: Dominik and 3 others" for example.

Reply:
Updated, thanks for the suggestion... I tried to implement this via Plural localized string but no luck, more info in internal discussion.